packages/server/modules/core/graph/resolvers/server.ts

```diff
@@ -41,7 +40,7 @@ module.exports = {
         .filter((role) => guestModeEnabled || role !== Roles.Server.Guest)
         .map((r) => ({
           id: r,
-          title: RoleInfo.Server[r]
+          title: RoleInfo.Server[r].title
```

Comment:
Moving to TS showed that there was an actual bug here - this resolver was broken and could not be executed

Reply:
![image](https://github.com/user-attachments/assets/b8934570-4214-4bc8-b85b-8568ed374252)